crates/accelerate/src/sparse_observable.rs

```diff
@@ -0,0 +1,1677 @@
+///                                    :attr:`indices` into complete terms.  For term number
+///                                    :math:`i`, its complex coefficient is ``coeffs[i]``, and its
+///                                    non-identity single-qubit operators and their corresponding
+///                                    qubits are the slice ``boundaries[i] : boundaries[i+1]`` into
+///                                    :attr:`bit_terms` and :attr:`indices` respectively.
+///                                    :attr:`boundaries` always has an explicit 0 as its first
+///                                    element.
+///   ==================  ===========  =============================================================
+///
+/// The length parameter :math:`t` is the number of terms in the sum, and the parameter :math:`s` is
+/// the total number of non-identity single-qubit terms.
+///
+/// As illustrative examples:
+///
+/// * in the case of a zero operator, :attr:`boundaries` is length 1 (a single 0) and all other
+///   vectors are empty.
+/// * in the case of a fully simplified identity operator, :attr:`boundaries` is ``[0, 0]``,
+///   :attr:`coeffs` has a single entry, and :attr:`bit_terms` and :attr:`indices` are empty.
+///
+/// These two cases are not special, they're fully consistent with the rules and should not need
+/// special handling.
+///
+/// The scalar item of the :attr:`bit_terms` array is stored as a numeric byte.  The numeric values
+/// are related to the symplectic Pauli representation that :class:`.SparsePauliOp` uses, and are
+/// accessible with named access by an enumeration:
+///
+/// ..
+///     This is documented manually here because the Python-space `Enum` is generated
+///     programmatically from Rust - it'd be _more_ confusing to try and write a docstring somewhere
+///     else in this source file. The use of `autoattribute` is because it pulls in the numeric
+///     value.
+///
+/// .. py:class:: SparseObservable.BitTerm
+///
+///     An :class:`~enum.IntEnum` that provides named access to the numerical values used to
+///     represent each of the single-qubit alphabet terms enumerated in
+///     :ref:`sparse-observable-alphabet`.
+///
+///     This class is attached to :class:`.SparseObservable`.  Access it as
+///     :class:`.SparseObservable.BitTerm`.
+///
+///     The numeric structure of these is that they are all four-bit values of which the low two
+///     bits are the (phase-less) symplectic representation of the Pauli operator related to the
+///     object, where the low bit denotes a contribution by :math:`Z` and the second lowest a
+///     contribution by :math:`X`, while the upper two bits are ``00`` for a Pauli operator, ``01``
+///     for the negative-eigenstate projector, and ``10`` for teh positive-eigenstate projector.
+///
+///     .. autoattribute:: qiskit.quantum_info::SparseObservable.BitTerm.X
+///
+///         The Pauli :math:`X` operator.
```

Comment:
It's sort of obvious but with the other labels mentioned below it might avoid some confusion to add it here (confusion as in: "doesn't Pauli X have a label too?")
```suggestion
///         The Pauli :math:`X` operator, labelled ``"X"``.
```

Reply:
Done in 9dd0187213.